oss-cloud-backend/handler.js

```diff
@@ -5,9 +5,11 @@ const utility = require('./services/utility.js');
 
 module.exports.getContributors = async (event) => {
   try {
+    console.log('Pre svega ikada');
```

Comment:
Please remove

Reply:
Unnecessary comments and console logs removed.